pkg/resources/statefulsets/kes-statefulset.go

```diff
@@ -116,9 +116,11 @@ func NewForKES(t *miniov2.Tenant, serviceName string) *appsv1.StatefulSet {
 	// External certificates will have priority over AutoCert generated certificates
 	if t.KESExternalCert() {
 		serverCertSecret = t.Spec.KES.ExternalCertSecret.Name
-		// This covers both secrets of type "kubernetes.io/tls" and
-		// "cert-manager.io/v1alpha2" because of same keys in both.
-		if t.Spec.KES.ExternalCertSecret.Type == "kubernetes.io/tls" || t.Spec.KES.ExternalCertSecret.Type == "cert-manager.io/v1alpha2" {
+		// This covers both secrets of type "kubernetes.io/tls",
+		// "cert-manager.io/v1alpha2" and "cert-manager.io/v1" because
+		// of same keys in both.
+		if t.Spec.KES.ExternalCertSecret.Type == "kubernetes.io/tls" || t.Spec.KES.ExternalCertSecret.Type == "cert-manager.io/v1alpha2" ||
+			t.Spec.KES.ExternalCertSecret.Type == "cert-manager.io/v1" {
```

Comment:
This additional check probably needs to be added too on the following files:
```
pkg/resources/jobs/kes-job.go:42
pkg/controller/cluster/kes.go:310
pkg/resources/deployments/console-deployment.go:225
pkg/resources/deployments/console-deployment.go:278
pkg/resources/deployments/console-deployment.go:314
pkg/resources/statefulsets/minio-statefulset.go:322
pkg/resources/statefulsets/minio-statefulset.go:369
pkg/resources/statefulsets/minio-statefulset.go:409
pkg/resources/statefulsets/minio-statefulset.go:369
pkg/resources/statefulsets/minio-statefulset.go:452
pkg/resources/statefulsets/minio-statefulset.go:369
pkg/resources/statefulsets/minio-statefulset.go:467
```
And all other places in where we check for `cert-manager.io/v1alpha2`

Reply:
Let me send a separate PR to add supported types. removing from this PR